package.json

```diff
@@ -4,7 +4,10 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "jest"
+    "test": "jest",
+    "lint": "eslint \"{api,tests}/**/*.ts\" --fix",
+    "build": "tsc",
```

Comment:
If I'm not mistaken. ts-jest runs the ts code without creating compiled js files. If so, we need to perform compilation without creating the js files. We don't need it